django/db/models/functions/__init__.py

```diff
@@ -5,6 +5,10 @@
     Now, Trunc, TruncDate, TruncDay, TruncHour, TruncMinute, TruncMonth,
     TruncQuarter, TruncSecond, TruncTime, TruncWeek, TruncYear,
 )
+from .math import (
+    Abs, Acos, Asin, Atan, Atan2, Ceiling, Cos, Cot, Degrees, Exp, Floor, Log,
```

Comment:
In [#28643 (comment:9)](https://code.djangoproject.com/ticket/28643#comment:9) there are a list of recommended names for these new functions which is already being followed in other PRs. Pay particular attention to capitalisation.

Please can you change to use `ACos`, `ASin`, `ATan`, `ATan2` and `Ceil` throughout - code, documentation, etc.